src/preferenceCenter/preferenceCenter.service.js

```diff
@@ -0,0 +1,226 @@
+import Logger from 'grapesjs-preset-mautic/src/logger';
```

Comment:
Here is some code from the dynamic content. I don't think that should be here?

Reply:
@disha-pishavadia24 can you take a look at the review please?

Reply:
@adiux @RCheesley Extra file removed. And that was not in consideration.